guide/additional-info/changes-in-v13.md

```diff
@@ -66,7 +66,7 @@ discord.js v13 makes the switch to Discord API v9! In addition to this, the new
 ## Slash commands
 
 discord.js now has support for slash commands!
-Refer to the [slash commands](/interactions/registering-slash-commands.html) section of this guide to get started.
+Refer to the [slash commands](/interactions/slash-commands.html) section of this guide to get started.
```

Comment:
```suggestion
Refer to the [slash commands](/interactions/slash-commands.md#registering-slash-commands) section of this guide to get started.
```

Reply:
It's the first section, is this really necessary?